File: Python/백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
    A = sorted(list(map(int,input().split())))
    if A[0] == A[1] == A[2] == 0:
        break
    elif A[0]**2 + A[1]**2 == A[2]**2:
        print('right')
    else :
        print('wrong')

File: Java/백준/Bronze/4153. 직각삼각형/직각삼각형.java
import java.io.IOException;
import java.util.*;

class Main {
    public static void main(String[] args) throws IOException {
        Scanner in = new Scanner(System.in);
        int[] A = new int [3];
        while (true){
            
            A[0] = in.nextInt();
            if (A[0]==0){
                break;
            }
            A[1] = in.nextInt();
            A[2] = in.nextInt();
            Arrays.sort(A);
            if (A[0]*A[0]+A[1]*A[1]==A[2]*A[2]){
                System.out.println("right");
            }
            else{
                System.out.println("wrong");
            }
        }
    }
}